src/main/java/org/example/db/DeliveryEmployeeDAO.java

```diff
@@ -1,14 +1,76 @@
 package org.example.db;
 
+import org.example.cli.DeliveryEmployee;
+import org.example.client.FailedToCreateException;
+
 import javax.swing.plaf.nimbus.State;
-import java.sql.Connection;
-import java.sql.ResultSet;
-import java.sql.SQLException;
-import java.sql.Statement;
+import java.sql.*;
 import java.util.ArrayList;
 import java.util.List;
 
 public class DeliveryEmployeeDAO {
+
+    // create instance of database connector class
+    DatabaseConnector databaseConnector = new DatabaseConnector();
+
+    /**
+     * inserts a new employee into employee table, then adds employee to delivery employee table via employee id
+     * @param deliveryEmployee
+     * @return
```

Comment:
What does it return?

Reply:
Changed the description to better reflect what the return type is